Lombiq.Privacy.csproj

```diff
@@ -45,8 +45,7 @@
     <None Remove="Tests\**" />
   </ItemGroup>
 
-  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Utilities\Lombiq.Npm.Targets\Lombiq.Npm.Targets.props" />
-  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Utilities\Lombiq.Npm.Targets\Lombiq.Npm.Targets.targets" />
+  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />

```

Comment:
I don't get any build or watch tasks now:

![image](https://user-images.githubusercontent.com/1976647/161110956-d065c3dd-59a3-48db-996f-767d3dd345bf.png)

(This is after a build.)

Reply:
You have to add them on a per-case basis, see [the docs on that](https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Docs/Styles.md#:~:text=in%20your%20package.json%2C%20add%20any%20or%20all%20of%20the%20following%20entries%20to%20the%20scripts%20property%3A%20).

Reply:
Yes.